project/plugins.sbt

```diff
@@ -29,3 +29,4 @@ addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.1.1")
 
+libraryDependencies += "javax.xml.bind" % "jaxb-api" % "2.3.1"
```

Comment:
this is a dependency override as riffraff artifact uses a java API (via AWS S3 client 1.11) that has been removed in JDK11.  Since most people use JDK8 this isn't strictly needed so I can remove or add a comment?